App/KeychainWrapper/Sources/KeychainWrapper/KeychainWrapper.swift

```diff
@@ -91,6 +92,21 @@ public extension KeychainWrapper {
             return false
         }
     }
+    
+    @discardableResult
+    func delete(key: String, accessibility: KeychainAttrRepresentable? = nil,  synchronizable: Bool = false ) -> Bool {
+        let query = queryDictionary(for: key, accessibility: accessibility, synchronizable: synchronizable)
+        
+        let status = SecItemDelete(query as CFDictionary)
+        
+        if(status == errSecSuccess || status == errSecItemNotFound) {
```

Comment:
something to consider: Maybe some methods from [SweetBool](https://github.com/sloik/SweetBool) could shorten or be more expressive here?